### DISCUSSION TITLE
Error importing pytorch lighting

### DISCUSSION THREAD

**Discussion by mshooter** [Lightning-AI/pytorch-lightning#6240] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hi, 
I am getting this weird error; I was able to run my code before and today I got this: 
import pytorch_lightning as pl                                              
"~/dir/miniconda3/envs/pytorchenv/lib/python3.7/site-packages/pytorch_lightning/__init__.py", line 66, in <module>
from pytorch_lightning import metrics                                       
ImportError: cannot import name 'metrics' from 'pytorch_lightning'

**Comment by akihironitta** [comment, marked as answer]
@mshooter Hi, could you try reinstalling it and running it again? I didn't experience the issue with the following command on Google Colab:
```python
!pip install pytorch-lightning --upgrade
from pytorch_lightning import metrics
```
 If the problem persists, could you run the following commands and share the output?
```console
$ wget https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master/tests/collect_env_details.py
$ python collect_env_details.py
```

**Reply by mshooter** [reply]
Hi @akihironitta I did re-install it and it's fine now! thanks for the help

**Reply by akihironitta** [reply]
@mshooter Glad it helped!
